docs/UsersGuide/source/BackgroundInfo/Introduction.rst

```diff
@@ -0,0 +1,181 @@
+   * :numref:`Section %s: Defining an SRW App Workflow <DefineWorkflow>` explains how to build a customized SRW App workflow XML file. 
+   * :numref:`Section %s: Template Variables <TemplateVars>` explains how to use template variables. 
+
+Reference Information
+-----------------------
+
+   * :numref:`Section %s: Rocoto Introductory Information <RocotoInfo>` provides an introduction to standard Rocoto commands with examples. 
+   * :numref:`Section %s: FAQ <FAQ>` answers users' frequently asked questions. 
+   * :numref:`Section %s: Glossary <Glossary>` defines important terms related to the SRW App. 
+
+
+SRW App Documentation Conventions
+===================================
+
+This guide uses particular conventions to indicate commands and code snippets, file and directory paths, variables, and options. 
+
+.. code-block:: console
+
+   Throughout the guide, this presentation style indicates shell commands, code snippets, etc.
+
+Text rendered as ``AaBbCc123`` typically refers to variables in scripts, names of files, or directories.
+
+Code that includes angle brackets (e.g., ``build_<platform>_<compiler>``) indicates that users should insert options appropriate to their SRW App configuration (e.g., ``build_hera_intel``). 
+
+File or directory paths that begin with ``/path/to/`` should be replaced with the actual path on the user's system. For example, ``/path/to/modulefiles`` might be replaced by ``/Users/Jane.Smith/ufs-srweather-app/modulefiles``. 
+
+Component Documentation
+=========================
+
+A list of available component documentation is shown in :numref:`Table %s <list_of_documentation>`. In general, technical documentation will explain how to use a particular component, whereas scientific documentation provides more in-depth information on the science involved in specific component files. 
+
+.. _list_of_documentation:
+
+.. list-table::  Centralized list of documentation
+   :widths: 20 50
+   :header-rows: 1
+
+   * - Documentation
+     - Location
+   * - HPC-Stack Documentation
+     - https://hpc-stack.readthedocs.io/en/latest/
+   * - spack-stack Documentation
+     - https://spack-stack.readthedocs.io/en/latest/
+   * - UFS_UTILS Technical Documentation
+     - https://noaa-emcufs-utils.readthedocs.io/en/latest
+   * - UFS_UTILS Scientific Documentation
+     - https://ufs-community.github.io/UFS_UTILS/index.html
+   * - UFS Weather Model User's Guide
+     - https://ufs-weather-model.readthedocs.io/en/latest
+   * - FV3 Technical Documentation
+     - https://noaa-emc.github.io/FV3_Dycore_ufs-v2.0.0/html/index.html
+   * - FV3 Scientific Documentation
+     - https://repository.library.noaa.gov/view/noaa/30725
+   * - CCPP Technical Documentation
+     - https://ccpp-techdoc.readthedocs.io/en/latest/
+   * - CCPP Scientific Documentation
+     - https://dtcenter.ucar.edu/GMTB/v6.0.0/sci_doc/index.html
```

Comment:
@mkavulich 2 questions!

1. Do these docs pertain to `develop` too? 
2. Is there any way to change the path name to include `UFS_SRW_App_v2.2.0` instead of `UFS_SRW_App_v3.0.0`? With the downgrading from 3.0 to 2.2, we may want to save that path for a future release...

Reply:
Good point, yes the page can be moved easily. It should be updated now to 2.2.0.